test/cppsuite/test_harness/workload/random_generator.cxx

```diff
@@ -37,25 +42,37 @@ random_generator::instance()
 }
 
 std::string
-random_generator::generate_string(std::size_t length)
+random_generator::generate_string(std::size_t length, characters_type type)
 {
-    std::string random_string;
+    std::string str;
 
-    for (std::size_t i = 0; i < length; ++i)
-        random_string += _characters[_distribution(_generator)];
+    switch (type) {
+    case characters_type::ALPHABET:
+        str = _alphabet;
+        break;
+    case characters_type::PSEUDO_ALPHANUMERIC:
+        str = _pseudo_alphanum;
+        break;
+    default:
+        testutil_die(type, "Unexpected characters_type");
+        break;
+    }
```

Comment:
The switch/case above is a duplicate of `random_generator::get_characters()`. We can call `std::string str = get_characters(type);` here.

Reply:
Why I did not reuse `get_characters` is because `get_characters` returns a reference and `shuffle` will modify it while I want `shuffle` to modify a copy.
If we modify the reference, it will impact the algorithm in `generate_pseudo_random_string`.
In order to reuse `get_characters`, I would need to make it return a copy. Do you think I should proceed this way ?

Reply:
Well, you need a copy of the string anyway. I guess we can do this `std::string str(get_characters(type));`?